types/datatables.net/datatables.net-tests.ts

```diff
@@ -35,7 +35,7 @@ const lang: DataTables.LanguageSettings = {
 
 //#region "Column"
 
-const colCreatedCellFunc: DataTables.FunctionColumnCreatedCell = (cell, cellData, rowData, rowIndex, colIndex) => {};
+const colCreatedCellFunc: DataTables.FunctionColumnCreatedCell = (cell, cellData, rowData, rowIndex, colIndex,cellls) => {};
```

Comment:
`cellls` is misspelled. Try correcting to `cells` to see if that fixes the build error.